test/Polly.Core.Tests/CircuitBreaker/Controller/ScheduledTaskExecutorTests.cs

```diff
@@ -56,22 +48,20 @@ public async Task ScheduleTask_Multiple_EnsureExecutionSerialized()
         using var verified = new ManualResetEvent(false);
 
         using var scheduler = new ScheduledTaskExecutor();
-        scheduler.ScheduleTask(
+        var task = scheduler.ScheduleTask(
             () =>
             {
                 executing.Set();
                 verified.WaitOne();
                 return Task.CompletedTask;
-            },
-            ResilienceContextPool.Shared.Get(CancellationToken),
-            out var task);
+            });
 
         executing.WaitOne();
 
-        scheduler.ScheduleTask(() => Task.CompletedTask, ResilienceContextPool.Shared.Get(CancellationToken), out var otherTask);
+        var otherTask = scheduler.ScheduleTask(() => Task.CompletedTask);
 
 #pragma warning disable xUnit1031 // Do not use blocking task operations in test method
-        otherTask.Wait(50, CancellationToken).ShouldBeFalse();
```

Comment:
Why revert these cancellation token usages? The field was added intentionally to reduce churn when we can adopt xunit v3 (#2440) as `CancellationToken.None` will be replaced with `TestContext.Current.CancellationToken`.